### DISCUSSION TITLE
Installation error

### DISCUSSION THREAD

**Discussion by mnishikawara** [PMEAL/OpenPNM#2941] [Q&A]
<img width="1920" alt="sc2024-10-28 103450" src="https://github.com/user-attachments/assets/d820bd0a-e23c-4936-b77a-8b4e99ee86ea">

I followed the installation guide by typing the following at the conda prompt: (conda install -c conda-forge openpnm)
https://openpnm.org/installation.html
After typing "y" to start installation, the following error happened. The screenshot is attached.

--
ERROR conda.core.link:_execute(951): An error occurred while installing package 'conda-forge::openssl-3.3.2-h2466b09_0'.
Rolling back transaction: done

LookupError('unknown encoding: utf-16-le')
--

How can I solne the problem?

Masa

**Comment by mnishikawara** [comment]
After old Anaconda was installed, I could install openpnm successfully.
The version is follows.
[Anaconda3-2024.02-1-Windows-x86_64.exe](https://repo.anaconda.com/archive/Anaconda3-2024.02-1-Windows-x86_64.exe)

But I still have a problem.
When I type first line in Get started.

Following error appeared.
--
(base) C:\Users\mnish>import openpnm as op
'import' is not recognized as an internal or external command,
operable program or batch file.
--

Can you address the problem?

**Reply by jgostick** [reply]
Great that we solved one problem.  Now you need to learn some python :-)  .  The imports and code happen at a python promp, not a windows propmt.

**Reply by mnishikawara** [reply]
python prompt that you mentioned is Anaconda prompt? 
Anaconda prompt rejects import command.
The installation of openpnm is exactly completed.
Can you address the issue?

**Reply by mnishikawara** [reply]
I could run the example in Spyder.
Thank you.
